packages/typegpu/src/std/atomic.ts

```diff
@@ -0,0 +1,69 @@
+import type { atomicI32, atomicU32 } from '../data/wgslTypes';
+import { inGPUMode } from '../gpuMode';
+
+type AnyAtomic = atomicI32 | atomicU32;
+
+export function atomicLoad<T extends AnyAtomic>(a: T): number {
+  if (inGPUMode()) {
+    return `atomicLoad(&${a})` as unknown as number;
+  }
+  throw new Error('Atomic operations are not supported outside of GPU mode.');
+}
+
+export function atomicStore<T extends AnyAtomic>(a: T, value: number): void {
+  if (inGPUMode()) {
+    // biome-ignore lint/correctness/noVoidTypeReturn: <string-void duality>
+    // biome-ignore lint/suspicious/noConfusingVoidType: <^>
+    return `atomicStore(&${a}, ${value})` as unknown as void;
```

Comment:
We can get away with less Biome wrestling 🤼‍♂️
```suggestion
    // biome-ignore lint/correctness/noVoidTypeReturn: <string-void duality>
    return `atomicStore(&${a}, ${value})` as unknown as undefined;
```

Reply:
I don't even need to turn it from void to undefined as it turns out